storage/localstore/localstore.go

```diff
@@ -362,6 +412,113 @@ func (db *DB) po(addr chunk.Address) (bin uint8) {
 	return uint8(chunk.Proximity(db.baseKey, addr))
 }
 
+func (db *DB) ListPinnedFiles() {
```

Comment:
not sure if we need this. this was never defined as a requirement

Reply:
I remember discussing it. May be we have missed capturing it. Also it makes sense to list the current pinned hashes and it's sizes, as they will be useful for pinning nodes.